syntax = "proto3";

message TransactionRequest {
  string sender_account_number = 1;
  string receiver_account_number = 2;
  double amount = 3;
}

message TransactionResponse {
  bool success = 1;
  string message = 2;
}

service TransactionService {
  rpc SendMoney(TransactionRequest) returns (TransactionResponse);
}
